frontend/src/main.js

```diff
@@ -1,6 +1,10 @@
-import './assets/main.css'
-
 import { createApp } from 'vue'
 import App from './App.vue'
 
-createApp(App).mount('#generated-frontend')
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { faHome, faTrash} from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
```

Comment:
This should be in `package.json` and `yarn.lock`, just install it again with `yarn add <package name>`, and add those two files.